contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -1455,6 +1455,14 @@ export const Clipboard = new Resource({
   },
 });
 
+function deleteMoveTasks(change) {
```

Comment:
Hmm i'm not familiar with this system, but does this basically ensure we delete any move tasks once theyre done in some form?

Reply:
Yeah, pretty much - we are using the tasks as a messenger format pretty much to revert the move if it fails, but after that we can just auto delete it.